toolkit-test-app/src/main/java/com/esri/arcgisruntime/toolkit/test/bookmark/BookmarksRepository.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.esri.arcgisruntime.toolkit.test.bookmark
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import com.esri.arcgisruntime.loadable.LoadStatus
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.BookmarkList
+
+class BookmarksRepository(private val map: ArcGISMap) {
```

Comment:
Should we ommit the repository here for simplicity? I don't think it adds much for this simple test/demo.

Reply:
removed